src/policy/concrete.rs

```diff
@@ -248,6 +248,9 @@ impl<Pk: MiniscriptKey> Policy<Pk> {
     /// `[pk(A),pk(B),and(or(pk(C),pk(D)),pk(E)))]`. Each policy in the vector is compiled into
     /// the respective miniscripts. A Huffman Tree is created from this vector which optimizes over
     /// the probabilitity of satisfaction for the respective branch in the TapTree.
+    ///
+    /// Refer to [doc/Tr compiler.pdf] in the root of the repository to understand why such
```

Comment:
Ideally, we also want the source code of this uploaded somewhere where I can copy/edit it. In case, we find some errors in the document, there would be no way to edit it.

Can you create a new repo with the code for this? And also link the source here

Reply:
I've made the gist [here](https://gist.github.com/SarcasticNastik/9e70b2b43375aab3e78c51e09c288c89).